### ISSUE TITLE
Slack community link has expired (again)

### ISSUE THREAD

**Issue by julemand101** [flutter/website#7990]
### Page URL

https://flutter.dev/community

### Describe the problem

(Same issue as https://github.com/flutter/website/issues/7468 in the past)

If going to the provided link on the community page for "Community Slack":
https://fluttercommunity.dev/joinslack

The link will then forward to:
https://fluttercommunity.slack.com/join/shared_invite/zt-1jg49rms1-yu~m81iePmqjRiWw5Wimng#/shared-invite/error

Which says:
> This link is no longer active
>
> To join this workspace, you’ll need to ask the person who originally invited you for a new link.

### Expected fix

A new link for the Slack community server needs to be generated.

**Comment by sfshaza2** [comment]
@khanhnwin, @atsansone, this keeps happening. Is there another, more permanent solution?

**Comment by julemand101** [comment]
I want to say that the link are inactive (again? not sure if it got fixed at any point?) right now:
![image](https://user-images.githubusercontent.com/116227/217498030-25ab60c8-2e57-472f-9cb9-baf868a35156.png)

**Comment by khanhnwin** [comment]
The slack link automatically expires after a certain amount of time. A quick search shows that the only solution is to continually generate a new link. I'll forward this over to Kelvin to make him aware.

**Comment by julemand101** [comment]
@khanhnwin 
Should there be some more automated way to detect and fix this? I can make a script that checks daily if the link still works and if not, create a GitHub issue or put a comment here. But I hope there are better ways to handle this? :)

I can see Slack does have a limit on 400 people joining at a given link. So it is not enough to just put some fixed period in a calendar and check if the link is still working.

A "solution" would be to reconsider Slack since it does not seem they really want to have their platform to be used the way which Flutter tries to promote.

**Comment by khanhnwin** [comment]
Yeah, for sure! Appreciate the offer :) I was thinking we could take that idea a step further and implement a GitHub action could automate the checking to see if it's still valid and regenerate the the invite link if it's expired. But to be totally honest, I'm not sure who has ownership over the slack channel so that's the first place I'll have to dig. 

```A "solution" would be to reconsider Slack since it does not seem they really want to have their platform to be used the way which Flutter tries to promote.``` That's a really good point. I ran a quick search and saw a few other folks ran into the same issue too. We do have the Discord server which is already on the community page. Not sure of the 'right" solution here... @kkboateng

**Comment by sfshaza2** [comment]
Filip _used_ to own this, but I can't remember who owns it now. (But it's
someone external to Google.)

This is a huge pain and I _do_ prefer Slack over Discord, but maybe we
should consider removing Slack from our community page.

**Comment by loic-sharma** [comment]
![image](https://user-images.githubusercontent.com/737941/217605494-3259c43b-d724-4703-874a-f77825ee86ea.png)

**Comment by RandalSchwartz** [comment]
@slightfoot normally fixes this...

**Comment by slightfoot** [comment]
Hi @sfshaza2 long time, nice to see you. @khanhnwin nice to see you again too. 

I'd rather not remove it from the site as it has a wide popular community. Resolution would be better.

I've been trying to resolve this for sometime. It wasn't originally a problem. I've been looking into a automated solution. But slacks API is for paid accounts. Slack might offer a free package, or just access to the API or perhaps a fix. Does anyone have contacts over at Slack?

I was thinking about a semi-automated solution. Basically revolving around having a slack bot ping the link.

For the moment, I'll refresh it. And it would be good if it happens again before I can get the bot sorted someone can email me or message and it can be fixed instantaneously.  

Edit.. Link fixed. 

Do we want to keep this issue open to monitor the issue ongoing?
